docpad.coffee

```diff
@@ -66,43 +66,34 @@ module.exports =
 
     # The entire schedule
     schedule: [
-      name: 'Eduardo Sganzerla',
-      email: 'esganzerla@gmail.com',
-      photo: 'themes/yellow-swan/img/eduardo-sganzerla.jpg',
-      bio: 'Usuário de desenvolvimento web desde aproximademente os 8 anos, quando teve acesso ao antigo HPG, em sábados após as 14h. Isso levou a aprender programar um pouco de Javascript e PHP 4. Conseguiu libertar do desenvolvimento por um tempo, se formou em Administração de Empresas, mas em 2015 teve uma recaida e está viciado em desenvolvimento front-end.',
-      # company: '',
+      name: 'Rafael Peixe',
+      email: 'rafaelpeixe@gmail.com',
+      photo: 'https://s.gravatar.com/avatar/da6fb493cecc5765c18254b4331c5d8f?s=240',
+      bio: 'Analista de Sistemas na Meta e Formando em Ciência da Computação na Feevale',
+      company: 'Meta',
       link:
-        href: 'http://twitter.com/esganzerla',
-        text: '@esganzerla'
+        href: 'mailto:rafaelpeixe@gmail.com',
```

Comment:
Do not add e-mail here.

Just let it blank since he doesn't provided any.

Reply:
Done!